components/member-profile/contribution/contribution.module.scss

```diff
@@ -16,22 +16,17 @@
   justify-content: space-between;
 }
 
-.featureTitle,
-.featureLink a {
-  color: #041484 !important;
-}
-
 .featureDescription {
   margin-top: 12px;
   color: gray;
 }
 
-.featureLink {
+#featureLink {
```

Comment:
Any reason for using ids?

Reply:
SO I was facing some issues with specificity, since `ids` have higher specificity than `classes` I used it.
But I improved the css, please check now.